flake8_fine_pytest/watchers/data_definition.py

```diff
@@ -0,0 +1,20 @@
+import ast
+
+from flake8_fine_pytest.watchers.base import BaseWatcher
+
+
+class LongDataDefinitionWathcer(BaseWatcher):
+    error_template = (
+        'FP004 Test {filename} has data definition longer than {max_data_lenght} strings.'
+    )
+
+    def run(self) -> None:
+        max_data_lenght = self.options.max_number_line_data_definition
+        data_lenght = 0
+        for node in ast.walk(self.tree):
+            if isinstance(node, ast.Assign) and isinstance(node.end_lineno, int):
+                data_lenght += node.end_lineno - node.lineno
+        if data_lenght > max_data_lenght:
+            self.add_error((0,
```

Comment:
Please, reformat this call to comply with styleguide: https://github.com/best-doctor/guides/blob/master/guides/en/python_styleguide.md#calls-formatting